sdk/identity/azure-identity/samples/README.md

```diff
@@ -0,0 +1,18 @@
+---
+page_type: sample
+languages:
+  - python
+products:
+  - azure
+  - azure-identity
+urlFragment: identity-samples
+---
```

Comment:
Looks like user needs to install keyvault to run these samples?

Maybe we should point it out in readme?